src/utilities/idf/WorkspaceObject.cpp

```diff
@@ -431,8 +431,11 @@ namespace detail {
     }
 
     // regular field -- name or data
-    if ((index == 0) && (iddObject().hasNameField())) {
```

Comment:
Could add additional check if this field is a handle here and route that to setName to preserve existing behavior.